src/components/MAPI/clusters/ClusterList/ClusterListItem.tsx

```diff
@@ -81,7 +81,7 @@ const ClusterListItem: React.FC<IClusterListItemProps> = ({
 }) => {
   const name = cluster?.metadata.name;
   const description = useMemo(() => {
-    if (!cluster || !providerCluster) return undefined;
+    if (!cluster || typeof providerCluster === 'undefined') return undefined;
```

Comment:
Do we need to do this in the cluster detail view, as well?

Reply:
Yes we do 😅